src/Numerics/ODESolvers/BackwardEulerSolvers.jl

```diff
@@ -67,6 +67,19 @@ solver.
 """
 update_backward_Euler_solver!(::AbstractBackwardEulerSolver, Q, α) = nothing
 
+"""
+    get_implicit_operator_coefficient(::AbstractBackwardEulerSolver)
+
+Returns the coefficient `α` of the implicit operator in
+an `AbstractBackwardEulerSolver` object:
+```
+    Q - α f(Q) = Qhat,
+```
+where `Q` is the state vector, `Qhat` is the right-hand side,
+and `f` is the discretized implicit operator.
+"""
+get_implicit_operator_coefficient(::AbstractBackwardEulerSolver) = nothing
```

Comment:
```suggestion
function get_implicit_operator_coefficient end
```
I think that this is the what we want so an error is thrown if the method is used but not defined for the concrete type

Reply:
Thanks!!